tests/unit/cn/vblock_builder_test.c

```diff
@@ -193,7 +199,7 @@ MTF_DEFINE_UTEST_PRE(test, t_vbb_finish_exact, test_setup)
     err = fill_exact(lcl_ti, vbb, 0, 0);
     ASSERT_EQ(err, 0);
 
-    err = vbb_finish(vbb, &blks);
+    err = vbb_finish(vbb, &blks, &max_kobj);
```

Comment:
One thing that I have done in the hblock_builder_test.c file is reading the mblock after writing it to confirm that the data I expect to be there is actually there. Might be nice to assert that you are storing the keys you think you are storing for instance.

Reply:
There are tests that reads back the vblock trailer and validates it.. but not all of them does.. Will see if I can add more of these validations.